### PR TITLE
fix building when only py3 is available in rhel8

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
Do we want to build only the Python 2, and not Python 3, packages on RHEL-7? I thought we'd want both there.

File: koji-containerbuild.spec [hunk 1]
```diff
@@ -36,12 +36,13 @@ Source0:        %{name}-%{version}.tar.gz
 BuildArch:      noarch
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:  python
-BuildRequires:  python2-devel
 %if 0%{with python3}
 BuildRequires:  python3-devel
-%endif
+BuildRequires:  python3-setuptools
+%else
```
Comment (MartinBasti) [inline koji-containerbuild.spec]: This means we stop building both py2 and py3 packages for Fedora, I don't think that we agreed on dropping py2 packages from fedora

Comment (MartinBasti) [reply koji-containerbuild.spec]: Was agreed that we drop py2 from fedora

**Comment by rcerven** [comment]
rn not updated, fix for unreleased change

**Pull request merged**